content-src/experiments/cliqz.yaml

```diff
@@ -96,5 +96,5 @@ contributors:
     display_name: 'Cliqz GmbH'
     avatar: /static/images/experiments/cliqz/avatars/cliqz-avatar.jpg
 installation_count: 1
-created: '2017-01-11T00:06:00.00Z'
+created: '2016-12-01T00:06:00.00Z'
```

Comment:
FYI to @6a68 I think we are going to do this so we don't have to rely on any late translations of "New" for our launch on Tuesday.